lib/fathead/mdn_css/output.txt

```diff
@@ -344,28 +389,151 @@
 :visited	A			css pseudo classes							<section class="prog__container"><p>The :visited CSS pseudo-class lets you select only links that have been visited. This style may be overridden by any other link-related pseudo-classes, that is :link, :hover, and :active, appearing in subsequent rules. In order to style appropriately links, you need to put the :visited rule after the :link rule but before the other ones, defined in the LVHA-order: :link — :visited — :hover — :active.</p><pre><code>:visited { style properties }</code></pre></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/:visited
-<angle>	A			css data types							<section class="prog__container"><p>The <angle> CSS data type represents angle values. Positive angles represent clockwise angles, negative angles represent counterclockwise angles. Its syntax is a <number> data type immediately followed by the unit (deg, grad, rad or turn). Like for any CSS dimension, there is no space between the unit literal and the number.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/angle
-<basic-shape>	A			css data types							<section class="prog__container"><p>The <basic-shape> type can be specified using basic shape functions. When using this syntax to define shapes, the reference box is defined by each property that uses <basic-shape> values. The coordinate system for the shape has its origin on the top-left corner of the reference box with the x-axis running to the right and the y-axis running downwards. All the lengths expressed in percentages are resolved from the used dimensions of the reference box.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/basic-shape
-<blend-mode>	A			css data types							<section class="prog__container"><p>The <blend-mode> type is a collection of keywords describing blend modes.</p><pre><code>Syntax not found in DB!</code></pre></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/blend-mode
-<color>	A			css data types							<section class="prog__container"><p>The <color> CSS data type denotes a color in the sRGB color space. A color can be described in any of these ways:using a keyword, using the RGB cubic-coordinate system (via the #-hexadecimal or the rgb() and rgba() functional notations), using the HSL cylindrical-coordinate system (via the hsl() and hsla() functional notations)</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/color_value
+<a>	R	:any-link										
+<angle>	A			css data types							<section class="prog__container"><p>An angle of direction for the gradient. See <angle>.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/repeating-linear-gradient#Values
```

Comment:
I believe the `<angle>` is going to break the page -- unfortunately we don't current escape content outside of the `<pre><code>` and we force any encoded text to be decoded, so this needs to be wrapped in `<code>`